var/spack/repos/builtin/packages/strumpack/package.py

```diff
@@ -70,7 +70,8 @@ class Strumpack(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("butterflypack@1.2.0:", when="@4.0.0: +butterflypack+mpi")
     depends_on("butterflypack@2.1.0:", when="@6.3.0: +butterflypack+mpi")
     depends_on("cuda", when="@4.0.0: +cuda")
-    depends_on("zfp", when="+zfp")
+    depends_on("zfp@0.5.5", when="@:7.0.1 +zfp")
+    depends_on("zfp@1.0.0", when="@7.0.2: +zfp")
```

Comment:
It could be `zfp@1.0.0:` if you want to make sure it is at least that version.

Reply:
Right, though strumpack@7.0.2 has been updated to handle any version of zfp.  I wouldn't require zfp@1.0.0, e.g., if it's not available.

Reply:
Come to think of it, how does spack handle dependencies on "any" version?  If, say, zfp 0.5.1 is already installed, will it match that rather than go find the latest version of zfp?

Reply:
The process to to arrive on a fixed set of package versions in a build is the concretization step.
It can be adjusted and e.g. `spack install --reuse` tries to concretize by reusing already installed or built dependencies.

AFAICS, you've resolved this review comment with your last comment.